api/interservice/authz/v2/project.proto

```diff
@@ -32,7 +32,9 @@ message GetProjectResp {
     Project project = 1;
 }
 
-message ListProjectsReq {}
+message ListProjectsReq {
+    bool globalFilterView = 1;
```

Comment:
This name describes the usage, not what it actually does. That is, if you want to use authz/ListProjects for the global projects filter, you set this flag. Alternately, could name it for what it does "alsoIncludeUnassignedWhereApplicable" if there is strong sentiment for this.

Reply:
So @srenatus has a comment below about unifying the terms "introspection" and "globalFilterView", since this is really need just for introspection. So as a third choice, we could rename this to just "introspection". Please vote for your choice here!